samples/apps/research-assistant-react/backend/README.md

```diff
@@ -0,0 +1,50 @@
+## FastAPI Backend for AutoGen Research Assistant
```

Comment:
There is inconsistency and overlap between the README in root of the app and the backend/ folder.